### ISSUE TITLE
Automatic initial estimate tool

### ISSUE THREAD

**Issue by sardnar** [pharmpy/pharmpy#2392]
### Feature request

Hello,

One feature that may enhance the algorithm is to set initial estimates automatically for parameters of interest. The idea is to have pre-fit for few subjects with no IIVs with FO or naive pooling. The selection of those subjects is probably going to be manual.

**Comment by rikardn** [comment]
I agree that this would be a nice addition. If you are using the `amd` tool the user currently needs to provide initial estimates for the starting model. All initial estimates for later models will be automatically generated.
